models/model_upload/llms/lmdeploy-llama-3_2-1b-instruct/1/model.py

```diff
@@ -0,0 +1,173 @@
+import os
+import subprocess
+import sys
+import threading
+from typing import Iterator
+
+from clarifai.runners.models.model_runner import ModelRunner
+from clarifai.utils.logging import logger
+from clarifai_grpc.grpc.api import resources_pb2, service_pb2
+from clarifai_grpc.grpc.api.status import status_code_pb2
+from google.protobuf import json_format
+from lmdeploy import pipeline, GenerationConfig, TurbomindEngineConfig
+from transformers import AutoTokenizer
+
+class MyRunner(ModelRunner):
+  """A custom runner that loads the model and generates text using vLLM Inference.
+  """
+
+  def load_model(self):
+    """Load the model here and start the vllm server."""
+    os.path.join(os.path.dirname(__file__))
+    # if checkpoints section is in config.yaml file then checkpoints will be downloaded at this path during model upload time.
+    checkpoints = os.path.join(os.path.dirname(__file__), "checkpoints")
+    # Note that Llama3.2-1B-Inst is not supported by turbomind yet, lmdeploy will auto switch to pytorch engine
```

Comment:
I think we should add model example that `lmdeploy` but we also should include this comment here for knowledge purpose

Reply:
The latest lmdeploy version now supports this model with the turbomind backend. 
I'm considering keeping a consistent model ID across all inference frameworks to easily compare performance differences. The PyTorch backend, being one of two in lmdeploy, is well-optimized as well.